### PR TITLE
Add search sorting

### PR DESCRIPTION
**Pull request by jessemyers**

### CODE REVIEW

File: microcosm_postgres/store.py [hunk 1]
```diff
@@ -160,9 +160,19 @@ def search(self, *criterion, **kwargs):
 
         """
         query = self._query(*criterion)
+        query = self._order_by(query)
```
Comment (srt32) [inline microcosm_postgres/store.py]: I don't understand the usage here. Could you give an example usage of this update?

Comment (srt32) [inline microcosm_postgres/store.py]: I don't understand the usage here. Could you give an example usage of this update?

Comment (jessemyers) [inline microcosm_postgres/store.py]: See next file in the tests/example.py

Comment (srt32) [inline microcosm_postgres/store.py]: I don't understand how the changes made here allow for that change to work

Comment (jessemyers) [inline microcosm_postgres/store.py]: The `_order_by` function is a mutator on the query object; by default is does nothing, but a subclass can write it to define whatever order by condition you want.

Comment (srt32) [inline microcosm_postgres/store.py]: ah, got it. Thanks.

**Comment by srt32** [comment]
LGTM

**Pull request closed without merging**